dominion/dominion-frontend/.gitignore

```diff
@@ -0,0 +1,69 @@
+# Logs
```

Comment:
These should be in the `HURUmap-apps/.gitignore`. I suggest we start using <https://gitignore.io> to create standard `.gitignore` files. In this case we should create one that supports both backend (python, Django, virtualenv, etc.) and frontend (NodeJS, React, etc.).